src/Microsoft.Azure.WebJobs.Host/Scale/InMemoryScaleMetricsRepository.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    public class InMemoryScaleMetricsRepository : IScaleMetricsRepository
```

Comment:
This is also a ScaleController specific implementation that would only be used by ScaleController so doesn't need to be in the SDK? I.e. it's not an implementation that needs to be shared.

Reply:
Agreed. `InMemoryScaleMetricsRepository` is moved to ScaleController repository.